datarobot_provider/example_dags/hospital_readmissions_xgboost_example.py

```diff
@@ -136,7 +138,7 @@ def hospital_readmissions_xgboost_example():
     get_blueprint_id = GetProjectBlueprintsOperator(
         task_id="get_blueprint_id",
         project_id=project_id,
-        model_type="xgboost",
+        filter_model_type="xgboost",
```

Comment:
Nice 👍 

I _just_ noticed this locally:
![Uploading Screenshot 2025-02-24 at 2.40.26 PM.png…]()